### DISCUSSION TITLE
DB++的热力图如何显示

### DISCUSSION THREAD

**Discussion by Li-Yidong** [PaddlePaddle/PaddleOCR#13879] [Q&A]
我用自己的数据集训练了一个DB++模型，请问如何在推理时输出热力图？

**Comment by UserWangZz** [comment]
需要自己在代码中可视化需要的部分

**Reply by Li-Yidong** [reply]
感谢您的回复！
请问具体需要改哪一部分的代码呢？

**Reply by UserWangZz** [reply]
可以具体说一下需要那部分呢？
主要的类名在config文件中都有涉及，具体代码在ppocr/路径下

**Reply by Li-Yidong** [reply]
我想输出论文中的threshold map, probability map和approximate map

**Reply by UserWangZz** [reply]
不好意思回复晚了，
尝试https://github.com/PaddlePaddle/PaddleOCR/blob/main/ppocr/modeling/heads/det_db_head.py 文件下查看代码进行可视化
